src/buildHTML.js

```diff
@@ -347,16 +347,18 @@ groupTypes.supsub = function(group, options, prev) {
     } else {
         supShift = Math.max(
             supShift, minSupShift,
-            sup.depth + 0.25 * fontMetrics.metrics.xHeight);
-        subShift = Math.max(subShift, fontMetrics.metrics.sub2);
+            sup.depth + 0.25 * options.style.metrics.xHeight);
+        subShift = Math.max(
+            subShift,
+            options.style.metrics.sub2);
```

Comment:
Why the line breaks? And why two distinct calls to `Math.max`, instead of including all of these into a single call?